tests/CoreFX/runtest/src/TestUtils/TestFileSetup/TestFileSetup.cs

```diff
@@ -0,0 +1,205 @@
+﻿using System;
```

Comment:
All source files must have a license header (like this one: https://github.com/dotnet/corert/blob/3781c96980e5959924f7d23c91682df43ddde1a6/samples/WebApi/Program.cs#L1)

In addition, please add comments describing what this program does and what kind of arguments it takes.